frontend/sw360-portlet/src/main/webapp/js/main.js

```diff
@@ -240,9 +240,11 @@ function deleteConfirmed(confirmMessage, confirmCallback) {
         content: confirmMessage,
         confirmButtonClass: 'btn-info',
         cancelButtonClass: 'btn-danger',
-        confirm: function() {
-            confirmCallback();
-        }
+        buttons: {
```

Comment:
This problem is now fixed with the lines [247ff](https://github.com/sw360/sw360portal/pull/369/commits/190402e6e0ac5dc526ebfc541ee4800004bfdb85#diff-b2d4b3e262a033e0e8c3a1a5e818d4ccR247)